src/thread_rng.rs

```diff
@@ -120,18 +120,15 @@ impl RngCore for ThreadRng {
 
 impl CryptoRng for ThreadRng {}
 
-/// DEPRECATED: use `thread_rng().gen()` instead.
-///
 /// Generates a random value using the thread-local random number generator.
 ///
 /// This is simply a shortcut for `thread_rng().gen()`. See [`thread_rng`] for
-/// documentation of the entropy source and [`Rand`] for documentation of
+/// documentation of the entropy source and [`Standard`] for documentation of
```

Comment:
Add a quick mention of using `thread_rng().gen_range(a, b)` here? That (and maybe a link to `Rng`) is enough though; we don't need to go over everything here.